instrumentation/aws-lambda/aws-lambda-core-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdacore/v1_0/AwsLambdaRequestHandlerInstrumentation.java

```diff
@@ -59,6 +60,8 @@ public void transform(TypeTransformer transformer) {
   @SuppressWarnings("unused")
   public static class HandleRequestAdvice {
 
+    private final static long flushTimeoutNanos = flushTimeout().toNanos();
```

Comment:
```suggestion
```

Reply:
I have intentionally matched the reference implementation from https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-java-instrumentation+flushTimeout.toNanos&type=code

Made it static as it is unlikely to change within the runtime, but I can make it non-static and non-final attribute if that will help.